unstructured/utils.py

```diff
@@ -35,7 +35,7 @@ def wrapper(*args, **kwargs):
                 raise ImportError(
                     f"Following dependencies are missing: {', '.join(missing_deps)}. "
                     + (
-                        f"Please install them using `pip install unstructured[{extras}]`."
+                        f"""Please install them using `pip install "unstructured[{extras}]"`."""
```

Comment:
was there a reason for the triple quote here? or some auto-formatting thing happen?

Reply:
Yes. `unstructured[blabla]` needs to have double quotes around it in the error message for the user to install correctly. If the quotes are not there, then it does not pip install.

Reply:
I guess you could probably use single quotes instead of triple double...

Reply:
ohhh totally didn't even see the quotes around `unstructrued[{extras}]`. YES, 100% nice catch!